tests/unit/ExampleTest.php

```diff
@@ -0,0 +1,22 @@
+<?php 
+class ExampleTest extends \Codeception\Test\Unit
+{
+    /**
+     * @var \UnitTester
+     */
+    protected $tester;
+    
+    protected function _before()
```

Comment:
WARNING: Method name "_before" should not be prefixed with an underscore to indicate visibility